kolibri/core/lessons/viewsets.py

```diff
@@ -97,3 +101,26 @@ def annotate_queryset(self, queryset):
         return annotate_array_aggregate(
             queryset, lesson_assignment_collections="lesson_assignments__collection"
         )
+
+    @action(detail=False)
+    def size(self, request, **kwargs):
+        lessons = self.get_queryset()
+        if lessons:
+            lessons_nodes_list = [{lesson.id: lesson.resources} for lesson in lessons]
+            lessons_sizes_dict = {}
```

Comment:
Should move this outside of the `if` statement - this is not defined unless `lessons` is truthy.